db/migrate/20230911002612_create_category_entities.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategoryEntities < ActiveRecord::Migration[7.0]
+  def change
+    create_table :category_entities do |t|
+      t.references :category, null: false, foreign_key: true
+      t.references :entity, null: false, foreign_key: true
+      t.timestamps
+    end
+  end
+end
```

Comment:
As a requirement for this milestone, kindly do well to follow the database structure as shown in the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png) diagram😉. The database should contain table for `users`, `entity` and `categories`. but I noticed you added an extra table `categories_entity`. please let's try to stick to the database structure as shown in the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png) diagram and ensure the `has_many` and `belongs_to` relationships are implemented👍🏼.
<img width="927" alt="image" src="https://github.com/SergioPeralta22/Budget-App/assets/64643995/c8af4cc2-88ae-4a5e-94e1-fb180c41cfe7">

Reply:
@Ibizugbe Here we have a many-to-many relationship. Can you please tell me how to create a many-to-many relationship without an auxiliary table? I don't know how to do it any other way, and all the documentation I have consulted suggests an auxiliary table. even if I were to use the has_and_belongs_to_many command in rails that would create an auxiliary table.
![image](https://github.com/SergioPeralta22/Budget-App/assets/78930134/892a1027-abde-4cbb-bef5-e8410a55c1ee)